liquid-rust-tests/tests/neg/assert_terminators/test-check.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(register_tool)]
+#![register_tool(lr)]
+#![feature(custom_inner_attributes)]
+#![lr::assert_behavior(check)]
```

Comment:
@atgeller I was thinking that it may be better to have a different syntax for these configurations, I was thinking something like:

```
#![lr::cfg(assert="check")]
```

I'm not quite sure how this attribute syntax works though.

Reply:
Sounds good, are you thinking a comma'ed list of `key=value`, and then validation would be performed when processing the attributes, or a comma'ed list of configuration settings, each with their own grammar object? Either way you would have to check for duplicates. In the first case error-handling would have to be done manually, but you only have to maintain it in one place (`collector.rs`, or some sub-file). In the second case, error-handling would be done automatically, but you have to maintain the syntax as well as the collector.

Reply:
Rust have attributes which are a comma separated list of `key=value` so I'd use that. My hunch is that the values here are typically simple (something you'd expect as a cli argument) so no need for complex parsing. Consider also that the current implementation also needs to check for duplicates, you can specify the same crate attribute multiple times.

Reply:
I've figured out how to reuse the rustc tools for this, but it'll be a bit messy at first. I'm trying to make sure the code I write is mostly reusable for a generic cfg, but it's slow going. May not finish today.